src/scss/initial/_scale.scss

```diff
@@ -21,7 +22,7 @@ html {
   @include tools.tokens--(scale.$theme-sizes);
   @include tools.tokens--(scale.$page-sizes);
 
-  --extra: #{(tools.size('wide-page') - tools.size('page')) / 2};
+  --extra: #{math.div(tools.size('wide-page') - tools.size('page'), 2)};
   --main-min: calc(100vw - var(--page-margin) * 2);
```

Comment:
Doesn't need to happen in this branch necessarily, but these seems like something that could move to the page-sizes map?

Reply:
@mirisuzanne I made a new card to take care of this. if `map-compile` gets merged into this branch soon I can look into relocating those variables afer that's done.